### PR TITLE
delet user

### PR DESCRIPTION
**Pull request by mrrenjithnair**
trying

### CODE REVIEW

File: delet user.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+yhe kya bolti tu
```
Comment (pawangspandey) [inline delet user.txt]: you should write any english sentence.

**Review by pawangspandey — COMMENTED** [review]
Renjith I have seen your code which is not looking good. please check my suggestion.

File: delet user.txt (hunk 1 above)
Comment (sumitjr) [reply delet user.txt]: approve him

Comment (pawangspandey) [reply delet user.txt]: ok

**Review by sumitjr — APPROVED** [review]
fine dude

**Pull request merged**